docsrc/credits_changelog.dxx

```diff
@@ -73,12 +73,34 @@
 
       <li> David St&ouml;ckel contributed the 3D convex hull functionality.
 
+      <li> lnw contributed fixes for C++ compilers in 2025.
+
+      <li> parona-source contributed improvements to regex in python.
```

Comment:
@parona-source not sure if you want to include your full name. Please make suggestions

Reply:
You can include my name. I go by Alfred Wingate.